fortran/testpar/async.F90

```diff
@@ -1313,12 +1314,15 @@ PROGRAM async_test
      IF(.NOT.registered)THEN
         ! No async compatible VOL found
         async_enabled = .FALSE.
+        IF(mpi_rank==0) WRITE(*,'(A,/)') "NATIVE"
      ELSE
+        IF(mpi_rank==0) WRITE(*,'(A,/)') "DAOS"
         CALL H5Vlregister_connector_by_name_f("daos", vol_id, hdferror)
         CALL check("H5Vlregister_connector_by_name_f", hdferror, total_error)
      ENDIF
 
   ELSE
+     IF(mpi_rank==0) WRITE(*,'(A,/)') "ASYNC"
```

Comment:
What if the connector under test isn't one of these connectors?

Reply:
Then it prints Native

Reply:
Will add flags API to get this information.

Reply:
In this PR or in another one?

Reply:
This PR.